src/convertToObject.js

```diff
@@ -6,7 +6,32 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const mainObj = {};
+
+  const workArr = sourceString.split(';');
```

Comment:
The variable name `workArr` is not very descriptive. Consider renaming it to reflect its contents or purpose, such as `styleDeclarations`.